backend/tests/test_admin_portale_alloggi_routes.py

```
@@ -0,0 +1,123 @@
+
+from models import Role, User
+from routes.admin_routes import PORTALE_CREDENTIALS_NOT_CONFIGURED, USER_NOT_FOUND, admin_bp
+from database import Base, SessionLocal, engine
+
+
+@pytest.fixture(scope="module")
+def app():
+    """Create a Flask app for admin Portale Alloggi route tests."""
+    flask_app = Flask(__name__)
+    flask_app.config["TESTING"] = True
+    flask_app.config["JWT_SECRET_KEY"] = "test-secret"
```

Ruff (0.15.0)

[error] Possible hardcoded password assigned to: "JWT_SECRET_KEY"

(S105)